README.md

```diff
@@ -4,6 +4,8 @@ The 'master' branch of this repository holds the canonical version of the HackSo
 
 The constitution is available in [Constitution.md](./Constitution.md).
 
+A PDF copy of the constitution and declaration can be produced by running `make` ([pandoc](https://pandoc.org/) is required). The declaration should be filled in with the approriate names, but not committed.
```

Comment:
After installing pandoc, I got the following error:

```console
% make
pandoc -s Constitution.md Declaration.md -o Constitution.pdf
pdflatex not found. Please select a different --pdf-engine or install pdflatex
make: *** [Constitution.pdf] Error 47
```

pdflatex should probably also be mentioned as a prerequisite

Reply:
added in 544317a56030fcb0e8579d0582fd0f2a140ed8fd